gapic-generator-cloud/templates/cloud/client/method/def/_bidi.erb

```diff
@@ -4,27 +4,35 @@
 <%= indent method.doc_description, "# " %>
 #
 <%- end -%>
-# @param requests [Enumerable<<%= method.request_type %> | Hash>]
+# @param requests [Google::Gax::StreamInput, Enumerable<<%= method.request_type %> | Hash>]
```

Comment:
Could you add a test for the other type(s)?

Reply:
Yes.